### PR TITLE
Fixes for the DUT serial handling with the HUP suite

### PR DESCRIPTION
**Pull request by robertgzr**

### CODE REVIEW

**Review by vipulgupta2048 — COMMENTED** [review]
Thanks for creating this PR @robertgzr Can you add the change to 2 other places where we use URLs as this would help in preventing similar errors in the future.
 
1. Line 141
2. Line 459

File: worker/lib/workers/testbot.ts [hunk 1]
```diff
@@ -147,6 +147,9 @@ class TestBotWorker extends EventEmitter implements Leviathan.Worker {
 			if (this.networkCtl != null) {
 				await this.networkCtl.teardown();
 			}
+
+      unlink(dutSerialPath);
```
Comment (robertgzr) [inline worker/lib/workers/testbot.ts:151]: @vipulgupta2048 I noticed that the local serial log file was super long... we need to do something like this to clean it up between runs I guess

Comment (vipulgupta2048) [reply worker/lib/workers/testbot.ts:151]: Yo, I was just going to ask you this. How long (size-wise) are we talking about for the logs? Also, regarding "between runs" do you mean like between suites? or between 1 Hup to another

Comment (robertgzr) [reply worker/lib/workers/testbot.ts:151]: @vipulgupta2048 between client runs, I guess it just concat's the DUT serial logs as long as the worker container stays around

Comment (vipulgupta2048) [reply worker/lib/workers/testbot.ts:151]: That seems about right. I am checking serial logs for a multi-suite that happens on the same device and there's subtle differences at the start. I was checking this https://jenkins.dev.resin.io/job/leviathan-raspberrypi4-64/856/ LGTM to have this

Comment (vipulgupta2048) [reply worker/lib/workers/testbot.ts:151]: @rcooke-warwick What do you think

**Comment by robertgzr** [comment]
@vipulgupta2048 @rcooke-warwick does this look ok?

**Comment by rcooke-warwick** [comment]
@balena-ci rebase

**Pull request closed without merging**